tests/golden/control-plane/appcat/appcat/20_role_vshn_redis_restore.yaml

```diff
@@ -10,7 +10,7 @@ rules:
   - apiGroups:
       - vshn.appcat.vshn.io
     resources:
-      - vshnredis
+      - '*'
```

Comment:
is this necessary in this PR? Seems like we changed the permissions here for nothing